charts/jaeger-v2/Chart.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v2
+appVersion: 1.53.0
```

Comment:
what is the meaning of this version? Is this a reference to Jaeger version? Should it be hardcoded like that?

Reply:
The appVersion is only informational according to Helm https://helm.sh/docs/topics/charts/#the-appversion-field  but it is used to mark the version of image tag if tag is not set.  You can see here (several places) for example [query-deploy.yaml](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger/templates/query-deploy.yaml#L55C28-L55C39) calls [query.image](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger/templates/_helpers.tpl#L636) which uses [renderImage](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger/templates/_helpers.tpl#L557) which defaults the image to use ```Chart.AppVersion```.  This ultimately is also used in the [bitnami common](https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_images.tpl#L9) chart that is referenced on the line below.

So YES it should be hardcoded (AND UPDATED!), but it can be removed IF replaced by tag in the ```image:``` section.  But given Jaeger uses a single binary for all deploys, this is a good global place to ensure they are all up to the same version.